index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="#">
+          <img src="images/Union.png" alt="menubar" />
+        </a>
+      </nav>
+    </header>
+    <main>
+      <section class="Port">
+        <div hreh="#" class="Menu"></div>
+        <h1>
+          Hey there!<br />
+          I'm Michael
+        </h1>
+
+        <h2>
+          I'm a Software<br />
+          Developer
+        </h2>
+
+        <p>
+          I can help you build a product,<br />
+          feature or website. Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don't
+          hestiate to contact me.
+        </p>
+
+        <ul id="SocMediaHand">
+          <li>
+            <a href="https://www.linkedin.com/in/lawrence-amoafo-90b57213a/"
+              ><ion-icon name="logo-linkedin"></ion-icon>
+            </a>
+          </li>
+          <li>
+            <a href="https://github.com/Mike47ip"
+              ><ion-icon name="logo-github"></ion-icon>
+            </a>
+          </li>
+          <li>
+            <a href="https://twitter.com/Mike_pee47"
+              ><ion-icon name="logo-twitter"></ion-icon>
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <ion-icon name="logo-facebook"></ion-icon>
+            </a>
+          </li>
+        </ul>
```

Comment:
Your design
![image](https://github.com/Mike47ip/mobilePortfolioWeb/assets/78251927/144f2f23-c031-4201-8198-17c6a658b014)

According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md), we must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

I would suggest you download all the icons from Figma and use them in your design 👍

Figma
![image](https://github.com/Mike47ip/mobilePortfolioWeb/assets/78251927/473a3582-81c6-44be-bdff-01994daafeba)